core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/CreatingBillingProjectMonitor.scala

```diff
@@ -110,11 +125,18 @@ trait CreatingBillingProjectMonitor extends LazyLogging {
 
           case operations: Seq[RawlsBillingProjectOperationRecord] if operations.forall(rec => rec.done && rec.errorMessage.isEmpty) =>
             // all operations completed successfully
-            gcsDAO.completeProjectSetup(project) map {
-              case util.Failure(t) =>
-                logger.info(s"Failure completing project for $project", t)
-                project.copy(message = Option(t.getMessage))
-              case Success(_) => project.copy(status = CreationStatuses.Ready)
+            for {
+              ownerGroupEmail <- samDAO.syncPolicyToGoogle(SamResourceTypeNames.billingProject, project.projectName.value, SamProjectRoles.owner).map(_.keys.headOption.getOrElse(throw new RawlsException("Error getting owner policy email")))
```

Comment:
a) this is easiest way to get the email
b) it is safer in case prior code changes
c) when we deploy this change there may be project creations in flight in an unexpected state